dynamic-test-driver/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "dynamic-test-driver",
+  "version": "1.0.0",
+  "description": "A test driver in nodeJS that runs SEND requests against the mocked UPS data",
+  "main": "test-driver.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Jose Miguel Gallas Olmedo",
+  "license": "MIT",
```

Comment:
Apache license! 

also, not sure about the author tag... author is always a topic... that's hot discussed in the community. SCM tools do show all the authors of the file.

Perhaps we do AeroGear team